libs/redux/userGuardians/api.ts

```diff
@@ -0,0 +1,14 @@
+import clientApi from '../clientApi';
+
+const userGuardiansApi = clientApi.injectEndpoints({
+  endpoints: (build) => ({
+    getuserGuardians: build.query({
+      query: () => `/terrain`,
+      providesTags: ['userGuardiansApi'],
+    }),
+  }),
+});
+
+export const { useGetuserGuardiansQuery } = userGuardiansApi;
+
+export default userGuardiansApi;
```

Comment:
Just to keep all redux APIs in the same naming convention.

Can you change `userGuardiansApi` to `UserGuardiansAPI`?